developers/weaviate/concepts/cluster.md

```diff
@@ -47,7 +47,9 @@ All of the scenarios below assume that - as sharding or replication is increased
 
 ### Advantages when increasing sharding
 * Run larger datasets
-* Speed up imports
+* Speed up imports.
+
+To use multiple CPUs efficiently, enable sharding when you import data. For the fastest imports, enable sharding even on a single node.
```

Comment:
I think sharding is always "on", as each collection always includes one or more shards. So the terms "enable sharding" might be confusing. Propose to remove this sentence.

Reply:
I think this means you need to have multiple shards to take advantage of multiple CPUs. I'll ask for clarification.